scripts/lib/api/htmlToMd.ts

```diff
@@ -43,376 +39,17 @@ export async function sphinxHtmlToMarkdown(options: {
   baseSourceUrl: string;
   releaseNotesTitle: string;
 }): Promise<HtmlToMdResult> {
-  const { html, url, imageDestination, baseSourceUrl, releaseNotesTitle } =
```

Comment:
Most of this function is replaced by `processHtml`, which returns structured data.